### PR TITLE
[office-js][office-js-preview] (Excel) Pulling latest Excel type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Release URL: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.debug.js
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Review by AlexJerabek — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -27632,7 +27691,8 @@ declare namespace Excel {
         * Represents the type of this cell value.
         *
         * @remarks
-        * [Api set: ExcelApi 1.16]
+        * [Api set: ExcelApi BETA (PREVIEW ONLY)]
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:27694]: Wait, how is this moving out of a named requirement set?

**Review by AlexJerabek — CHANGES_REQUESTED** [review]

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:27694]: I don't think this and the following two API set tag changes count as a regression. 

These three have always been within the LocalImageCellValue object, which has always been tagged as BETA: https://learn.microsoft.com/en-us/javascript/api/excel/excel.localimagecellvalue?view=excel-js-preview

I think they were just tagged incorrectly as 1.16 at some point. They were only ever released in the preview office.d.ts (not the prod office.d.ts), and they aren't listed in the 1.16 release article: https://learn.microsoft.com/en-us/javascript/api/requirement-sets/excel/excel-api-1-16-requirement-set?view=common-js-preview.

Comment (AlexJerabek) [reply types/office-js-preview/index.d.ts:27694]: Sounds good. Thanks for verifying.

**Review by AlexJerabek — APPROVED** [review]

**Comment by alison-mk** [comment]
Ready to merge

**Pull request merged**